orchard-ws/app/controllers/WorkflowController.scala

```diff
@@ -135,6 +135,18 @@ class WorkflowController @Inject() (
       Future.successful(Results.Unauthorized(JsNull))
   }
 
+  def delete(workflowId: String) = authAction.async {
+    case ValidApiRequest(apiRole, req) =>
+      db.orchardDB
+        .async(new WorkflowQuery(workflowId).deletePending())
+        .map { r =>
+          if (r >= 1) Ok(Json.toJson(r))
+          else NotFound(Json.toJson(s"Pending workflow ${workflowId}} does not exist"))
```

Comment:
@realstraw one too many `}` for `${workflowId}`?

Reply:
good catch, fixed